src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
```

Comment:
does this PR cover and complete this (https://github.com/Samsung/TizenFX/pull/2876) ?
then could you close it?

Reply:
I will rebase this PR when https://github.com/Samsung/TizenFX/pull/2876 is merged.